### PR TITLE
[ALLUXIO-3254] Journal the correct mount id.

### PR DESCRIPTION
**Pull request by calvinjia**
https://alluxio.atlassian.net/browse/ALLUXIO-3254

### CODE REVIEW

File: minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java [hunk 1]
```diff
@@ -211,6 +211,16 @@ public void stopFS() throws Exception {
     stopMasters();
   }
 
+  /**
+   * Formats the master and then restarts it. This is useful if a fresh state is desired, for
```
Comment (aaudiber) [inline minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java]: Javadoc sounds like the formatting happens before the stopping

File: tests/src/test/java/alluxio/server/ft/journal/JournalCheckpointIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+
+/**
+ * Tests checkpoints remain consistent with intended master state.
+ */
+public class JournalCheckpointIntegrationTest {
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/server/ft/journal/JournalCheckpointIntegrationTest.java]: extend base integration test

File: minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java (hunk 1 above)
Comment (calvinjia) [reply minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java]: Updated, thanks.

File: tests/src/test/java/alluxio/server/ft/journal/JournalCheckpointIntegrationTest.java (hunk 2 above)
Comment (calvinjia) [reply tests/src/test/java/alluxio/server/ft/journal/JournalCheckpointIntegrationTest.java]: Updated, thanks.

**Pull request merged**